dev-qt/qtnetworkauth/qtnetworkauth-5.9999.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit qt5-build
+
+DESCRIPTION="Network authorization component library for the Qt5 framework"
+LICENSE="GPL-3"
```

Comment:
yeah I was asking about the license. You're right though, qtnetworkauth seems to be GPL3-only, unlike most other qt modules.